maintainers/maintainer-list.nix

```diff
@@ -15560,6 +15560,10 @@
     githubId = 7669898;
     name = "Katharina Fey";
   };
+  spalf = {
+    email = "tom@tombarrett.xyz";
+    name = "tom barrett";
```

Comment:
How about adding the `github` and `githubId` attributes ?

Reply:
i'd honestly prefer being contacted by email if that's okay

Reply:
`githubId` and `github` is helpful as then you'll be automatically requested as a reviewer in github, when someone does a PR on one of your packages

Reply:
@70m6 Awaiting for your feedback on this before merging the PR.

Reply:
```suggestion
    name = "tom barrett";
    github = "70m6";
    githubId = 105207964;
```

Reply:
> i'd honestly prefer being contacted by email if that's okay

You will likely only ever receive pings via GitHub with some rare exceptions.